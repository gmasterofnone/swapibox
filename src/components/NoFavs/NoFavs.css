.no-favs {
  color: #feda4a;
  font-family: 'Pathway Gothic One', sans-serif;
  font-size: 3rem;
  margin-top: 100px;
}