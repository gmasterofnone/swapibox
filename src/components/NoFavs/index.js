import React from 'react';

import './NoFavs.css';

const NoFavs = () => {
  return (
    <div className='card-container'>
      <h1 className='no-favs'>PLEASE ADD SOME FAVORITES!</h1>
    </div> 
  )
}

export default NoFavs;


