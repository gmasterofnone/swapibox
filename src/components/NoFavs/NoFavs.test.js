import React from 'react';
import { shallow } from 'enzyme';
import NoFavs from '.';

describe('NoFavs', () => {
  
  it('matches the snapshot', () => {
    let wrapper = shallow(<NoFavs />);

    expect(wrapper).toMatchSnapshot();
  });

});