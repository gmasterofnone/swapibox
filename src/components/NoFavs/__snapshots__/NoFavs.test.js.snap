// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoFavs matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NoFavs />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1
        className="no-favs"
      >
        PLEASE ADD SOME FAVORITES!
      </h1>,
      "className": "card-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "PLEASE ADD SOME FAVORITES!",
        "className": "no-favs",
      },
      "ref": null,
      "rendered": "PLEASE ADD SOME FAVORITES!",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1
          className="no-favs"
        >
          PLEASE ADD SOME FAVORITES!
        </h1>,
        "className": "card-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "PLEASE ADD SOME FAVORITES!",
          "className": "no-favs",
        },
        "ref": null,
        "rendered": "PLEASE ADD SOME FAVORITES!",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
